test/e2e/hive_manager.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Hive manager", func() {
+	Describe("CreateNamespace", func() {
+		var ns *corev1.Namespace
```

Comment:
maybe instead of having two describes you could just have something like this 
```suggestion
var _ = Describe("Hive manager creates a namespace", func() {
		var ns *corev1.Namespace
```